exp/controllers/helpers.go

```diff
@@ -477,7 +477,7 @@ func MachinePoolToInfrastructureMapFunc(gvk schema.GroupVersionKind, log logr.Lo
 
 // AzureClusterToAzureMachinePoolsFunc is a handler.MapFunc to be used to enqueue
 // requests for reconciliation of AzureMachinePools.
-func AzureClusterToAzureMachinePoolsFunc(ctx context.Context, kClient client.Client, log logr.Logger) handler.MapFunc {
+func AzureClusterToAzureMachinePoolsFunc(ctx context.Context, cli client.Client, log logr.Logger) handler.MapFunc {
```

Comment:
curious was this a suggestion from the linter? Minor but I kind of like the additional information from having the k while reading the code.

Reply:
Yes, the `revive` linter complained about this:

```go
exp/controllers/helpers_test.go:569:5: var-naming: don't use leading k in Go names; var kClient should be client (revive)
```

I agree that `kClient` is a more informative name than `cli`. I didn't use `client` because that's a package name already imported in those files (although we could import it with a different name).

I'm open to suggestions if there's a better var name to use here (that doesn't start with `k` apparently).

Reply:
don't have a great one, apparently it is just variables starting with k and upper case after: https://github.com/mgechev/revive/blob/76b8c5732985b4332321ab368730fc16810726a1/rule/var-naming.go#L94-L101

so something like k8sCleint would work but not sure how I actually feel about it. cli is probably fine, was more interesting the k leading variable thing 😄

Reply:
looking at the rest of the codebase https://github.com/kubernetes-sigs/cluster-api-provider-azure/search?q=client.Client, we're using `client`, `c`, `k8sClient`, `kubeClient` in different places. We should probably standardize on one instead of introducing yet another name for it.

Reply:
makes sense  to standardize, my vote would be for `kubeClient` or `k8sclient`

Reply:
I like `k8sClient` and `kubeClient`. I'll try to hunt down the references and standardize them.

Reply:
I just went with local convention, as I should have done initially. So that's `c` in helpers.go and `fakeClient` in helpers_test.go.

We do have a hodgepodge of variable names for this, and so does CAPI. Seems like `c` is the winner overall. I'm not sure it would be worth standardizing all of them; consistency is nice but maybe not *that* nice.